docker/src/main/java/org/jclouds/docker/compute/strategy/DockerComputeServiceAdapter.java

```diff
@@ -329,4 +319,19 @@ public void suspendNode(String id) {
       api.getContainerApi().pause(id);
    }
 
+   protected static Predicate<Image> createPredicateMatchingRepoTags(final String imageIdOrName) {
+      return new Predicate<Image>() {
+         @Override
+         public boolean apply(Image input) {
+            final String imageNameWithHostPrefix = "docker.io/" + imageIdOrName;
```

Comment:
Move "docker.io" to a constant that can be referenced from the tests too? IIRC previous versions of Docker prefixed official images with `library/`. I agree this is unlikely to change, but I think it is worth having that prefix isolated in a single constant.